src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), grid-template-columns should not have a fixed number of columns. Instead, use 'auto-fill' or 'auto-fit' to make the grid layout more flexible and responsive.